examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -308,19 +325,24 @@ sparkz {
         scId: "1f3b001136ca1afb15e13ac325170929feca6282bdfd89365684bf01f7c6ba14"
         powData: "1dd1b161f70e0f201dd1b161f70e0f201dd1b161f70e0f201dd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201bd1b161f70e0f201bd1b161f80e0f201bd1b161f80e0f201bd1b161f80e0f201bd1b161f80e0f201bd1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f20"
-        mcBlockHeight: 450
+        mcBlockHeight: 480
         mcNetwork: regtest
+        isNonCeasing: false
         withdrawalEpochLength: 900
         initialCumulativeCommTreeHash: "68ed60c2e761bca39e2acb0b97bc1d6cab3c4e34e9418e315a0a152c092da60e"
     }
 }
 ```
+_For non ceasing sidechains withdrawalEpochLength parameter has value of virtualEpochLength._
 
 Beside genesis related data backward transfer proof related data shall be updated as well. Similar to previous step append to configuration file next:
 `withdrawalEpochCertificate` block with next parameters:
- 1. `submitterIsEnabled`  - boolean. Enable/disable submitter to submit backward transfer certificate. Submitter uses zen-cli RPC calls.
- 2. `certificateSigningIsEnabled` - boolean (true by default). Enable/disable certificate signing for the Threshold Signature Circuit if node handles any of the signers private keys.
- 3. `signersPublicKeys` - list of Schnorr public keys were generated by command `generateCertificateSignerKey`. Please pay attention: order is important here! **Keys must be in the same order they were specified in generateCertProofInfo**, i.e. for example the input of `generateCertProofInfo` from step 6 next part of config is correct:
+1. `submitterIsEnabled`  - boolean. Enable/disable submitter to submit backward transfer certificate. Submitter uses zen-cli RPC calls.
+2. `certificateSigningIsEnabled` - boolean (true by default). Enable/disable certificate signing for the Threshold 
+Signature Circuit if node handles any of the signers private keys.
+3. `signersPublicKeys` - list of Schnorr signer public keys were generated by command `generateCertificateSignerKey`. 
+Please pay attention: order is important here! **Keys must be in the same order they were specified in generateCertProofInfo**, 
+i.e. for example the input of `generateCertProofInfo` from step 6 next part of config is correct:
```

Comment:
1. Please specify what file we should use as a template for KeyRotationCircuit case in the beginning of this step.
2. Describe how we should add Mster publick keys list to this configuration file.

Reply:
Done